desktop/main.ts

```diff
@@ -404,14 +413,13 @@ const mainWindow = () => {
                         submenu: [
                             {
                                 id: 'back',
-                                role: 'back',
                                 accelerator: process.platform === 'darwin' ? 'Cmd+[' : 'Shift+[',
                                 click: () => {
                                     browserWindow.webContents.goBack();
                                 },
                             },
                             {
-                                role: 'back',
+                                label: 'backWithKeyShortcut',
```

Comment:
Without `role`/`label` this item made the app crash, since 'back' role doesn't exist I've added a label, it's not shown since this item is not visible.